.github/actions/update-csv-bundles/action.yml

```diff
@@ -0,0 +1,173 @@
+name: "Update CSV bundles"
+description: "Generate and finalize OLM CSV bundles for a given version"
+
+inputs:
+  version:
+    description: "Version without v-prefix"
+    required: true
+  token:
+    description: "GitHub token to create PRs"
+    required: true
+  changelog-path:
+    description: "Path to the changelog file"
+    required: false
+
+outputs:
+  major_minor:
+    description: "Major.minor part of the version"
+    value: ${{ steps.vars.outputs.major_minor }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Derive release branch name
+      id: vars
+      shell: bash
+      run: |
+        VERSION="${{ inputs.version }}"
+
+        MAJOR_MINOR="${VERSION%.*}"
+        echo "major_minor=${MAJOR_MINOR}" >> "$GITHUB_OUTPUT"
+    - name: Checkout release branch
+      uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
+      with:
+        ref: release-${{ steps.vars.outputs.major_minor }}
+        fetch-depth: 0
+    - name: Setup Go
+      uses: actions/setup-go@4dc6199c7b1a012772edbd06daecab0f50c9053c #6.0.1
+      with:
+        go-version-file: "${{ github.workspace }}/go.mod"
+    - name: Setting properties
+      shell: bash
+      env:
+        CHANGELOG_CONTENT: ${{ inputs.changelog-path }}
+        DIGEST: ${{ inputs.digest }}
+        VERSION: ${{ inputs.version }}
+      run: |
+        VERSION="${VERSION}"
+
+        echo "Setting version to v${VERSION} in files"
+        ./.github/scripts/set_property.py --file "config/helm/chart/default/Chart.yaml" --property "appVersion" --value "v${VERSION}" 
```

Comment:
this could be replaced with `yq -i ".appVersion = \"$VERSION\"" config/helm/chart/default/Chart.yaml`. yq is [preinstalled on github hosted ubuntu runners](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#tools).